src/interfaces/IOptionSettlementEngine.sol

```diff
@@ -119,52 +195,115 @@ interface IOptionSettlementEngine {
     }
 
     struct Underlying {
+        // address of the underlying asset erc20
         address underlyingAsset;
+        // position on the underlying asset
         int256 underlyingPosition;
+        // address of the exercise asset erc20
         address exerciseAsset;
+        // position on the exercise asset
         int256 exercisePosition;
     }
 
-    // @notice The protocol fee, expressed in basis points
-    // @return The fee in basis points
+    /**
+     * @notice The balance of protocol fees for a given token which have not yet
+     * been swept.
+     * @param token The token for the unswept fee balance.
+     * @return The balance of unswept fees.
+     */
+    function feeBalance(address token) external view returns (uint256);
+
+    /**
+     * @notice The protocol fee, expressed in basis points.
+     * @return The fee in basis points.
+     */
     function feeBps() external view returns (uint8);
 
-    // @return The address fees accrue to
+    /**
+     * @notice Returns the address to which protocol fees are swept.
+     * @return The address to which fees are swept
+     */
     function feeTo() external view returns (address);
 
-    // @return The balance of unswept fees for a given address
-    function feeBalance(address token) external view returns (uint256);
-
-    // @return The enum (uint8) Type of the tokenId
+    /**
+     * @notice Returns the token type (e.g. Option/Claim) for a given token Id
+     * @param tokenId The id of the option or claim.
+     * @return The enum (uint8) Type of the tokenId
+     */
     function tokenType(uint256 tokenId) external view returns (Type);
 
-    // @return The optionInfo Option struct for tokenId
+    /**
+     * @notice Returns Option struct details about a given tokenID if that token is
+     * a vToken.
```

Comment:
Is vToken a thing? This is the first time i'm seeing it. Maybe a definition on what exactly that is would be helpful for future readers.

Reply:
Same here, new term to me. @Flip-Liquid what do you mean here by vToken?

Reply:
Heh, new in that I forgot about it, apparently. I think I meant "valid valorem token" i.e. not None type.

Reply:
fixed